chainlit_app/app.py

```diff
@@ -0,0 +1,66 @@
+import services.chat_service
+import services.feedback_service
+import services.history_service
+from chainlit import Message, on_chat_start, on_message
+
+
+@on_chat_start
+async def chat_start() -> None:
+    """
+    Triggered when the chat starts. Welcomes the user
+    and gives options to fetch history.
+    """
+    await Message(
+        content="Welcome! You can ask questions on health services delivery."
+    ).send()
+
+
+@on_message
+async def main(message: Message) -> None:
+    """
+    Handles user messages and provides a response from the chat API.
+    """
+    user_message = message.content
+
+    response = services.chat_service.get_chat_response(user_message)
+    if "error" in response:
+        await Message(content=f"Error: {response['error']}").send()
+    else:
+        answer = response.get("answer", "No answer found.")
```

Comment:
Why would this ever happen? Also, if we use a pydantic class, we can run validation automatically on `response`